### PR TITLE
Kubelet QPS postmortem

### PR DESCRIPTION
**Pull request by hjacobs**
This is just a mini-postmortem for the Kubelet QPS problem causing CDP build errors.
I mentioned this incident in my recent Kubernetes Failure Stories talk at ContainerDays: https://www.slideshare.net/try_except_/kubernetes-failure-stories-or-how-to-crash-your-cluster-containerdays-eu-2019

### CODE REVIEW

File: docs/postmortems/jun-2019-kubelet-qps.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+![CDP Build Errors](images/jun-2019-kubelet-qps/build-errors.png)
+
+CDP builds were timing out and failing because Kubernetes builder Pods couldn’t get AWS IAM credentials.
+For the Pod to get AWS IAM credentials, [kube2iam](https://github.com/jtblin/kube2iam) has to get the Pod’s IP address. These are set by kubelet process on the assigned node, but Kubelet was delayed in updating Pod statuses for multiple minutes.
+The default Kubelet configuration has an extremely low rate limit on the requests to the API server (see [--kube-api-qps documentation](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)).
+Due to an unlucky rescaling of the CDP Kubernetes cluster,
+only one node was available for the builder Pods, and the rapid creation and deletion of Pods caused Kubelet to fall behind.
+
+To stop the bleeding, the cluster was manually scaled up so builder Pods would be spread over more than 1 nodes.
```
Comment (mikkeloscar) [inline docs/postmortems/jun-2019-kubelet-qps.md]: more than 1 **node**

Comment (hjacobs) [reply docs/postmortems/jun-2019-kubelet-qps.md]: fixed

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**